api/pkg/apis/v1alpha1/providers/states/k8s/k8s_test.go

```diff
@@ -102,6 +102,7 @@ func TestActivationUpsert(t *testing.T) {
 					Stage:    "s1",
 				},
 			},
+			ETag: "1",
```

Comment:
Per offline discussion, there's no need to pass etag in creation case. Let's skip it.